src/API/Helpers/State/DummyStateHandler.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Auth0\SDK\API\Helpers\State;
+
+/*
+ * This file is part of Auth0-PHP package.
+ *
+ * (c) Auth0
+ *
+ * For the full copyright and license information, please view the LICENSE file
+ * that was distributed with this source code.
+ */
+
+/**
+ * Dummy implementation of the StateHandler
+ *
+ * @author Auth0
+ */
+class DummyStateHandler implements StateHandler 
+{
```

Comment:
`class` braces style is different from the `method` one?

Reply:
Shouldn't be but not consistent in the library (yet)